Add docs for service accounts 8.5 (#46801)
grafana/grafana: docs/sources/administration/api-keys/about-api-keys.md

---
title: About API keys in Grafana
menuTitle: About API keys
description: 'Learn about using API keys in Grafana'
weight: 30
---

# About API keys in Grafana

> A API Key is a randomly generated string that is used to interact with the Grafana HTTP APIs.

Reviewer: ```suggestion
An API key is a randomly generated string that external systems use to interact with Grafana HTTP APIs.
```

> When you create an API key, you specify a **Role** that determines permissions. For more information about creating API keys, refer to [Create an API key]({{< relref "./create-api-key.md#">}}).

Reviewer: ```suggestion
When you create an API key, you specify a **Role** that determines the permissions associated with the API key. Role permissions control that actions the API key can perform on Grafana resources. For more information about creating API keys, refer to [Create an API key]({{< relref "./create-api-key.md#">}}).
```

---
title: About API keys in Grafana
menuTitle: About API keys
aliases: [docs/sources/administration/api-keys/about-api-keys.md]

> description: 'This contains detailed information about API keys in Grafana'

Author: you meant Learn here right?

> description: 'This contains detailed information about API keys in Grafana'

Author: ```suggestion
description: 'Learn about using API keys in Grafana'
```

weight: 30
---

# About API keys in Grafana

> An API key is a encrypted string that identifies as a specific role to your Grafana instance when talking to apis. You can more easily use Grafana HTTP APIs by leveraging API keys.

Reviewer: Update this after you've updated the _index file.

To create an API key, use the following procedure. An API key is valid for a limited time that you specify when you create it, up to 30 days.

> When you create an API key, you specify a role for the key. The role determines the level of administrative power that users of the key have. For more information about creating API keys, refer to [Create API key]({{< relref "./create-api-key.md#">}}).

Reviewer: What are the levels? I would be interested to know more about the types of keys that a user can generate, and what they would be used for. This is a great place to add two or three examples.

> When you create an API key, you specify a role for the key. The role determines the level of administrative power that users of the key have. For more information about creating API keys, refer to [Create API key]({{< relref "./create-api-key.md#">}}).

Reviewer: ```suggestion
When you create an API key, you specify a role that determines which permissions the key has. For more information about creating API keys, refer to [Create API key]({{< relref "./create-api-key.md#">}}).
```
